main.py

```diff
@@ -1,10 +1,72 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
+def create_or_update_race(race_data: dict) -> Race:
```

Comment:
`create_or_update_race`? What exactly are you updating here?

Reply:
if there is a race with the same name that already exists but with a different description, the description will be updated, if I correctly understood how the method get_or_create() works.